tests/chart_tests/test_logging_sidecar.py

```diff
@@ -42,3 +44,50 @@ def test_logging_sidecar_config_disabled(self, kube_version):
             show_only="templates/logging-sidecar-configmap.yaml",
         )
         assert len(docs) == 0
+
+    def test_logging_sidecar_custom_config(self, kube_version):
+        """Test logging sidecar config with customConfig flag enabled"""
+        custom_side_carconfig = textwrap.dedent(
+            """
+        loggingSidecar:
+          enabled: true
+          customConfig: true
+          name: sidecar-logging-consumer
+          airflowSidecarConfig: |2
```

Comment:
What is this `|2` syntax? I can't find any info on it, and it looks like it behaves the same as `|`

<https://yaml-multiline.info>

Reply:
It denotes the block indentation number.  As you said no change in behaviour